crates/types/src/invocation/mod.rs

```diff
@@ -902,13 +902,22 @@ impl SpanRelation {
     }
 }
 
+#[derive(
+    Debug, PartialEq, Eq, Clone, Hash, serde::Serialize, serde::Deserialize, bilrost::Message,
+)]
+pub struct IngressInvocationResponseSink {
+    #[bilrost(1)]
+    pub request_id: PartitionProcessorRpcRequestId,
+}
+
 /// Definition of the sink where to send the result of a termination, or of purge.
-#[derive(Debug, PartialEq, Eq, Clone, Hash, serde::Serialize, serde::Deserialize)]
+#[derive(
+    Debug, PartialEq, Eq, Clone, Hash, serde::Serialize, serde::Deserialize, bilrost::Oneof,
+)]
 pub enum InvocationMutationResponseSink {
     /// The invocation has been generated by a request received at an ingress, and the client is expecting a response back.
-    Ingress {
-        request_id: PartitionProcessorRpcRequestId,
-    },
+    #[bilrost(5)]
+    Ingress(IngressInvocationResponseSink),
```

Comment:
@muhamadazmy did anything change regarding enum variants with fields with the latest Bilrost update? Is it still required to create a wrapper type for the fields of the variant?

Reply:
Unfortunately yes. The support for enum variants with multiple fields is not yet supported. There is this issue here https://github.com/mumbleskates/bilrost/issues/4 on bilrost

Reply:
What do I need to do then? How about I revert this to just use serde? This command is not hot path.

Reply:
It's okay to revert this to use serde only for now. Once the bilrost issue is resolved I can go over the types again that can be migrated to bilrost without the need for wrapper/shadow types.